frontend/src/types/sql-review-schema.yaml

```diff
@@ -32,6 +32,7 @@ ruleList:
       - TIDB
```

Comment:
This file is related to SQL review feature. We can take out this file for now and have a separate PR to support SQL review for DM.

Reply:
Fixed.